popup.el

```diff
@@ -1,4 +1,4 @@
-;;; popup.el --- Visual Popup User Interface
+;;; popup.el --- Visual Popup User Interface -*- lexical-binding: t -*-
```

Comment:
The standard is two spaces after the description. 🤔 Can you change it for me? Thanks!